CLI/polymath-cli.js

```diff
@@ -140,20 +141,22 @@ program
   .alias('str')
   .description('Runs STR Migrator')
   .action(async function(toStrAddress, fromTrAddress, fromStrAddress) {
-    await strMigrator.executeApp(toStrAddress, fromTrAddress, fromStrAddress, program.remoteNode);
+    await strMigrator.executeApp(toStrAddress, fromTrAddress, fromStrAddress);
   });
 
 program
   .command('permission_manager')
   .alias('pm')
   .description('Runs permission_manager')
   .action(async function() {
-    await permission_manager.executeApp(program.remoteNode);
+    await permission_manager.executeApp();
   });
 
 program.parse(process.argv);
 
 if (typeof program.commands.length == 0) {
   console.error('No command given!');
   process.exit(1);
+} else {
+  global.constants.NETWORK = program.remoteNode
```

Comment:
Instead of this, maybe we could call to global.Initialize(program.remoteNode) here. This way, we could remove initialize command from everywhere and call it only once.

Reply:
Good idea! I'll figure out how could I implement it.

Reply:
Is it possible to replace all the occurrences of 'Initialize()' for a single line here? I don't remember how Commander works, but it could work.

Reply:
it's not possible. We haven't the ability to run an action before the command catcher. I had to modify every action to perform the initialize method outside the module controller.